DayTomato/DayTomato.Droid/Util/ClusterRenderer.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using Android.Gms.Maps.Model;
+using Com.Google.Maps.Android.Clustering.View;
+using Com.Google.Maps.Android.Clustering;
+using Android.Gms.Maps;
+using Android.Content;
+using Java.Lang;
+using Com.Google.Maps.Android.UI;
+using Android.Widget;
+using Android.Graphics;
+using Android.Graphics.Drawables;
+
+namespace DayTomato.Droid
+{
+    
+
+	public class ClusterRenderer : DefaultClusterRenderer
+	{
```

Comment:
Please make sure naming convention is followed, instead of mVariable, should be _variable, and that private static is worrying me as well